stories/src/examples/MarkerFilters.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { ReactNode, useMemo, useState } from 'react'
+import { Label, Checkbox, themeColor } from '@datapunt/asc-ui'
+import styled from 'styled-components'
+
+interface MarkerFiltersProps {
+  markers: Array<[number, number]>
+  clusterComponent: (markers: any) => ReactNode
+}
+
+const UnstyledOl = styled.ol`
+  margin: 0;
+  padding: 0;
+  list-style-type: none;
+`
+
+const Panel = styled.div`
+  background-color: ${themeColor('tint', 'level2')};
+  padding: 10px;
+  box-shadow: 0 0 0 5px rgba(0, 0, 0, 0.1);
+`
+
+const setCount = 6
+
+export default function MarkerFilters({
+  markers,
+  clusterComponent,
+}: MarkerFiltersProps) {
+  const [activeDatasets, setActiveDataset] = useState([0, 1, 2])
+
+  const markerSets = useMemo(() => {
+    const datasets = []
+    let i = 0
+
+    for (i; i < setCount; i += 1) {
+      datasets.push({
+        id: i,
+        title: `Dataset ${i + 1}`,
+        markers,
+        isActive: false,
+      })
+    }
+    return datasets
+  }, [markers])
+
+  const markersFiltered = useMemo(() => {
+    let i = 0
+    // eslint-disable-next-line no-shadow
+    let markersFiltered: any[] = []
```

Comment:
We could use `LatLngTuple` from leaflet as a type here

Reply:
Will do.